### PR TITLE
Story/2010

### PR DESCRIPTION
**Pull request by peterpratt13**

### CODE REVIEW

File: addons/udes_stock/README.md [hunk 1]
```diff
@@ -279,26 +279,26 @@ Model used to register users that can access the system in the way has been defi
 
 ### Picking Naming Convention
 
-If a backorder gets created from a picking, then it will either have `-001` appended if the prefix does not exist, else it will cause it to increment by one. How sequencing works is that the suffixed sequences refer to the level in the picking tree, not the nth backorder.
+If a backorder gets created from a picking, then it will either have `-001` appended if the prefix does not exist, else it will cause it to increment it by one. Sequencing works as if the tree is actually a chain, the suffix refers to the point of its creation, not the position in the tree.
 
-Example:
+Example (The Ref coulmn refers to the order of creation):
```
Comment (kdwyer) [inline addons/udes_stock/README.md:284]: s/coulmn/column

**Review by kdwyer — COMMENTED** [review]
Looks good.  Did you black-format the second commit?  Some of the formatting looks a bit off.

File: addons/udes_stock/tests/test_limit_orderpoints.py [hunk 2]
```diff
@@ -50,11 +52,13 @@ def test02_cannot_add_orderpoint_if_limit_on_parent(self):
...
-            self.create_orderpoint(self.banana, self.op_test_output_location_01, 5, 10)
+            self.create_orderpoint(
+                self.banana, self.op_test_output_location_01, 5, 10)
         ex = cm.exception
         self.assertEqual(
             "An order point for location {} already exists on "
-            "{}.".format(self.op_test_output_location_01.name, self.apple.name),
+            "{}.".format(self.op_test_output_location_01.name,
+                         self.apple.name),
             ex.name,
```
Comment (armandcela) [inline addons/udes_stock/tests/test_limit_orderpoints.py:62]: Shouldnt be change here as well deprecated name

**Review by armandcela — APPROVED** [review]

File: addons/udes_stock/tests/test_limit_orderpoints.py [hunk 3]
```diff
@@ -64,10 +68,12 @@ def test03_cannot_add_orderpoint_if_limit_on_grandparent(self):
...
```
[the remark this answers is not in the document]
Comment (peterpratt13) [reply addons/udes_stock/tests/test_limit_orderpoints.py]: Good spot - missed that in the logs

File: addons/udes_stock/tests/test_limit_orderpoints.py (hunk 2 above)
Comment (peterpratt13) [reply addons/udes_stock/tests/test_limit_orderpoints.py]: Good spot - missed that in the logs

File: addons/udes_stock/tests/test_limit_orderpoints.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (kdwyer) [reply addons/udes_stock/tests/test_limit_orderpoints.py]: We can consider replacing `format` with f-strings too.

**Comment by peterpratt13** [comment]
Force push to fixup commits and format

File: addons/udes_stock/tests/test_limit_orderpoints.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (peterpratt13) [reply addons/udes_stock/tests/test_limit_orderpoints.py]: this is why I shouldn't have done the drive by fix to clear up the logs - will do it now

**Review by kdwyer — APPROVED** [review]

**Review by PeterAlabaster — APPROVED** [review]

**Pull request merged**